plugins/modules/ovirt_disk.py

```diff
@@ -371,7 +372,7 @@ def transfer(connection, module, direction, transfer_func):
             time.sleep(module.params['poll_interval'])
             transfer = transfer_service.get()
 
-        if module.params['use_proxy']:
+        if module.params['use_proxy'] or not engine_supported(connection, '4.4'):
```

Comment:
What does engine_supported(connection, '4.4') mean?

The proxy should be used only in 2 cases:
- you cannot access the host, regardless of engine version
- the user ask to use the proxy

Reply:
This guarantees backwards compatibility. As far as I understood the Ansible code against oVirt <=4.3 so far did only know about proxy mode, that's why I didn't want to change that by default.